ts/sagas/startup.ts

```diff
@@ -69,6 +71,15 @@ import { watchWalletSaga } from "./wallet";
  */
 // tslint:disable-next-line:cognitive-complexity no-big-function
 function* initializeApplicationSaga(): IterableIterator<Effect> {
+  // FIXME: Workaround for iOS only. Below iOS version 12.3 Keychain is not
+  //        cleared between one installation and another, so it is needed to
+  //        manually clear previous installation user info in order to force
+  //        the user to choose PIN and run through onboarding every new
+  //        installation.
+  if (Platform.OS === "ios") {
+    yield call(previousInstallationDataDeleteSaga);
```

Comment:
on Android, `isFirstRunAfterInstall` will be `true` on each run (even after the first)